app/src/main/res/layout-sw600dp-land/onboarding_slide.xml

```diff
@@ -16,49 +16,43 @@
 
     <ImageView
       android:id="@+id/slide_image_view"
-      android:layout_width="0dp"
+      android:layout_width="wrap_content"
       android:layout_height="0dp"
       android:adjustViewBounds="true"
       android:contentDescription="@{viewModel.contentDescription}"
       android:scaleType="fitXY"
       android:src="@{viewModel.slideImage}"
       app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintDimensionRatio="H, 10:9"
```

Comment:
After following this approach -and setting the width to 0dp-, the thumbnails appeared horizontally compressed.  
## After using 
```
app:layout_constraintDimensionRatio="H, 10:9"
```
![screenshot-2020-05-31_11 12 41 093](https://user-images.githubusercontent.com/8533363/83348801-0c662580-a330-11ea-8eda-1654f5cccb60.png)
![screenshot-2020-05-31_11 12 55 522](https://user-images.githubusercontent.com/8533363/83348825-3ddef100-a330-11ea-896a-20b560c9035c.png)
## Before using
```
app:layout_constraintDimensionRatio="H, 10:9"
```
![screenshot-2020-05-30_12 37 12 047](https://user-images.githubusercontent.com/8533363/83348805-11c37000-a330-11ea-811a-e8ae372eb711.png)
![screenshot-2020-05-30_12 37 23 743](https://user-images.githubusercontent.com/8533363/83348841-4e8f6700-a330-11ea-9e14-4e934444194e.png)

And it is obvious that sticking with the approach that doesn't use `app:layout_constraintDimensionRatio="H, 10:9"` will give us a better result as it looks exactly the same as the [mocks](https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/00a93a92-aa03-42d4-9a98-d186eaeec2ca/L-Onboarding-4). But using `app:layout_constraintDimensionRatio="H, 10:9"` will make the images a bit compressed.  
The first approach will always look better as it uses `android:layout_height="0dp"` so that the image will always fill the screen vertically, and `android:layout_width="wrap_content"` which will make the image take whatever width it needs, this will make the image appear with the correct ratio because the image file itself is in the correct ratio. Finally, `android:scaleType="fitXY"` ensures that the image is in the right place.
@rt4914 PTAL

Reply:
Try `40:39` or `39:40` that should work because the image that you are using here follows that ratio.

Ratios are important ass they make sure that behaviour is consistent across devices. In this Screen this looks correct because mocks have been designed for this only.

Reply:
40:39 is perfect 👌  
![screenshot-2020-05-31_14 34 29 311](https://user-images.githubusercontent.com/8533363/83352492-0b42f180-a34c-11ea-98db-99d2794fed8e.png)
![screenshot-2020-05-31_14 35 16 889](https://user-images.githubusercontent.com/8533363/83352498-1433c300-a34c-11ea-9ec7-af541fd50704.png)
![screenshot-2020-05-31_14 35 40 305](https://user-images.githubusercontent.com/8533363/83352501-185fe080-a34c-11ea-93e8-c7fbe909c167.png)
![screenshot-2020-05-31_14 35 00 609](https://user-images.githubusercontent.com/8533363/83352496-10a03c00-a34c-11ea-9853-b5e593f0cd0e.png)

Reply:
Thanks.